scripts/aem.js

```diff
@@ -646,7 +646,7 @@ async function loadHeader(header) {
   const headerBlock = buildBlock('header', '');
   header.append(headerBlock);
   decorateBlock(headerBlock);
-  return loadBlock(headerBlock);
+  await loadBlock(headerBlock);
```

Comment:
This should also return the loaded block.

Reply:
@sdmcraft  this is already part of https://github.com/aemsites/icicidirect/pull/291 so I will remove this once https://github.com/aemsites/icicidirect/pull/291 gets merged.